test/schema/schema_test.go

```diff
@@ -215,6 +215,29 @@ func TestSchema_integration(t *testing.T) {
 		assert.Nil(t, errRm)
 	})
 
+	t.Run("CHECK /schema/{className}", func(t *testing.T) {
+		client := testsuit.CreateTestClient(8080, nil)
+
+		schemaClass := &models.Class{
+			Class: "Band",
+		}
+
+		err := client.Schema().ClassCreator().WithClass(schemaClass).Do(context.Background())
+		assert.Nil(t, err)
+
+		ok, err := client.Schema().ClassExistenceChecker().WithClassName("Band").Do(context.Background())
+		assert.Nil(t, err)
+		assert.True(t, ok)
+
+		ok, err = client.Schema().ClassExistenceChecker().WithClassName("NonExistantClass").Do(context.Background())
```

Comment:
NonExist**e**ntClass

Reply:
Oh right, I just copy pasted it from another test.